spec/actions/pod_push_with_error_handling.rb

```diff
@@ -0,0 +1,61 @@
+module Fastlane
+  module Actions
+    
+    class PodPushWithErrorHandlingAction < Action
+      def self.run(params)
+        begin
+          # Capture the output of pod_push
+          UI.message("🚀 Running pod_push with path: #{params[:path]}")
+          
+          output = Fastlane::Actions::PodPushAction.run(
+            path: params[:path],
+            synchronous: true 
```

Comment:
I would probably try to mimick the API of the `pod push` action. I think some of the options can be useful for this action as well, like `verbose` and `allow_warnings`... Not sure about others though. Wdyt?